### ISSUE TITLE
Show current road as a widget

### ISSUE THREAD

**Issue by jverkoey** [ClutchEngineering/Sidecar#78]
![IMG_5802](https://github.com/user-attachments/assets/ab89b4c3-dc6e-4b5d-91f9-81745c36673b)

**Comment by natebe4** [comment]
yes this! I use sygic maps primarily due to this feature. I love being able to see what road i’m on even when not navigating. even my cars default nav has this and it is a great feature. highly recommended

**Comment by Jomino1** [comment]
I second this, it’d be a great addition!
